pkg/bpmn_engine/engine_timer_test.go

```diff
@@ -27,6 +27,24 @@ func TestEventBasedGatewaySelectsPathWhereTimerOccurs(t *testing.T) {
 	then.AssertThat(t, cp.CallPath, is.EqualTo("task-for-message"))
 }
 
+func TestInvalidTimer(t *testing.T) {
+	// setup
+	bpmnEngine := New("name")
+	cp := CallPath{}
+
+	// given
+	process, _ := bpmnEngine.LoadFromFile("../../test-cases/message-intermediate-invalid-timer-event.bpmn")
+	bpmnEngine.AddTaskHandler("task-for-timer", cp.CallPathHandler)
+	instance, _ := bpmnEngine.CreateAndRunInstance(process.ProcessKey, nil)
+
+	// when
+	bpmnEngine.PublishEventForInstance(instance.GetInstanceKey(), "message")
+	bpmnEngine.RunOrContinueInstance(instance.GetInstanceKey())
```

Comment:
Here should be some assertion, which checks the presence of the error.

Reply:
> Here should be some assertion, which checks the presence of the error.

I have no way to catch the error because of handleElement does  not return this error.

Reply:
fair point.